parser.go

```diff
@@ -249,6 +252,18 @@ func (p *Parser) parseCreateTable(ctx *parseCtx) (model.Table, error) {
 		return nil, errors.New(`expected TABLE`)
 	}
 
+	// IF NOT EXISTS
+	ctx.skipWhiteSpaces()
+
+	var notexists bool
+	if ctx.peek().Type == IF {
+		ctx.advance()
+		if _, err := p.parseIdents(ctx, NOT, EXISTS); err != nil {
+			return nil, err
+		}
+		notexists = true
+	}
+
```

Comment:
https://dev.mysql.com/doc/refman/5.6/ja/create-table.html

maybe we should parse ```temporary``` first

Reply:
I agree